CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Help/ViewControllers/HelpFormatsViewController.swift

```diff
@@ -44,6 +45,9 @@ class HelpFormatsViewController: UIViewController {
     }
 
     private func configureMainView() {
+        if UIDevice.current.userInterfaceIdiom == .pad {
+            horizontalMargin = 126 - 16
```

Comment:
@chriskryn can we use constants here something like:
private let marginX = 126
horizontalMargin = marginX - margin ?😊